src/Microsoft.DotNet.Build.Tasks/PackageFiles/IL.targets

```diff
@@ -5,12 +5,13 @@
   <Target Name="CreateManifestResourceNames" Condition="'@(EmbeddedResource)' != ''" />
 
   <Target Name="GetIlasmPath"
-          Condition="'$(BuildingInsideVisualStudio)' == 'true'">
+          Condition="'$(OS)' == 'Windows_NT' And '$(IlasmToolPath)' == ''">
```

Comment:
Yeah if I hook the restore/publish directly into init-tools, then I can just eliminate this whole target and always use the one under Tools.